### DISCUSSION TITLE
Webview2 et DarkReader

### DISCUSSION THREAD

**Discussion by bilou47190** [darkreader/darkreader#8988] [Q&A]
Bonjour,
J'ai écrit un programme en VB.NET qui se sert de Webview2, Puis-je mettre DarkReader dans mon programme ?

Hello,
I have written a program in VB.NET that uses Webview2, can I put DarkReader in my program ?

**Comment by Gusted** [comment]
If webview2 uses a engine like Chromium that supports WebExtension, then it would be possible. But to implement that, would require some work.

**Comment by bilou47190** [comment]
Ok, je vais faire des recherches. Merci beaucoup.

Okay, I'll do some research. Thank you very much.
